app/Pages/Home.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet, View, Image } from 'react-native';
+import SavedWalls from '../Components/SavedWalls'
+import CreateNew from '../Components/CreateNew'
+
+export default function Home() {
+  return (
+    <View style={styles.viewContainer}>
+      <View style={styles.imageContainer}>
+        <Image
+          source={require('../assets/Klime.png')}
+          style={styles.image}
+        />
```

Comment:
Great job on the icon Hollis! 🧗Looks good!
I have 2 comments which are subjective so let's chat as a team before changing anything.
1. I think the icon is taking up too much screen real estate and that it can probably be a little smaller. Maybe we can have a loading state that displays the image and a loading bar? (STRETCH!!)
2. Do we always want the logo visible? If so I think it makes more sense having it in the App.js component. TBD

Reply:
Thanks for making the app logo Hollis! It looks great!